### DISCUSSION TITLE
Horizontal and Verticle ScrollableFrame orientation

### DISCUSSION THREAD

**Discussion by DominicCulyer** [TomSchimansky/CustomTkinter#1339] [General]
Is it possible to have a horizontal and vertically oriented scrollable frame?

**Comment by alfmelchor** [comment]
Check the wiki page for CTkScrollableFrame :)! One of the options is orientation, which allows you to choose between vertical and horizontal.
https://github.com/TomSchimansky/CustomTkinter/wiki/CTkScrollableFrame

**Reply by DominicCulyer** [reply]
I had a look, but it seems I can only have one or the other. Is there a way I can have both?

**Reply by MaxNiark** [reply]
nop, but you can get width autoscaling from screening, so yview it's not necessary
